tests/roles/run_tests/vars/main.yml

```diff
@@ -162,7 +162,7 @@ pivot_actions:
     - "pivoting"
     - "upgrading"
 repivot_actions:
-    - "ci_test_provision"
+    - "ci_test_repivoting"
```

Comment:
this is confusing to me as well. You are introducing `ci_test_repivoting` here but just line below there is `repivoting` as well

Reply:
Previously it was` ci_test_provision`, just to keep it similar I have added the `ci_test_repivoting`. It allows to run the fetch log from target cluster before re-pivot. It used only in integration test.

Reply:
Thanks for the discussion. I am reusing the `repivoting` action.